src/pages/home/report/ReportActionCompose.js

```diff
@@ -64,25 +64,16 @@ class ReportActionCompose extends React.Component {
         this.comment = props.comment;
 
         this.state = {
-            isFocused: false,
+            isFocused: true,
             textInputShouldClear: false,
             isCommentEmpty: props.comment.length === 0,
             isMenuVisible: false,
         };
     }
 
     componentDidUpdate(prevProps) {
-        // The first time the component loads the props is empty and the next time it may contain value.
-        // If it does let's update this.comment so that it matches the defaultValue that we show in textInput.
-        if (this.props.comment && prevProps.comment === '' && prevProps.comment !== this.props.comment) {
```

Comment:
The first time the component is mounted `this.comment` is attributed in the `constructor(){...}` of this component